web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx

```diff
@@ -52,12 +54,12 @@ export function CliCommand({ cliCommand, onRun, isLoading }: CliCommandProps) {
         mr="2"
         color={shouldDisplayIsLoading ? 'text.secondary' : 'text.primary'}
         width="100%"
+        style={{ fontFamily: theme.fonts.unsanitizedMono }}
```

Comment:
I'm not sure if we want to customize all occurrences of the mono font within the app. This has the potential of breaking the UI simply because the text will differ in width or actual height etc.

I think I'd rather keep the customized font only for the terminal, unless there are reasons for customizing all mono fonts that I don't see.

Reply:
+1 - this should only change the terminal font

Reply:
Good idea, so we will have the following fonts in the theme:
- fonts.sansSerif,
- fonts.mono,
- fonts.terminal (this one will be configurable).

Reply:
I don't know why I thought about adding that terminal font to theme, it doesn't make much sense as we want to use it only in one place.